src/main/java/org/fundacionjala/coding/manuel/Kata1.java

```diff
@@ -0,0 +1,45 @@
+package org.fundacionjala.coding.manuel;
+
+import java.util.StringJoiner;
+
+/**
+ * Created by Administrator on 2/22/2018.
+ */
+public class Kata1 {
+    private String sentence;
+    private static final int MAX_SIZE = 5;
+    private static final String SPACE =  " ";
+    /**
+     *@param w Constructor by default.
+     */
+    public Kata1(String w) {
+        this.sentence = w;
+    }
+    /**
+     *@return Method to spin the sentence or word.
+     */
+    public String spinWord() {
+        String[] words = sentence.split(" ");
+        String newSentece = "";
```

Comment:
Do you really need newSentece variable ?
you can use your StringJoiner j variable.